arangod/GeneralServer/SslServerFeature.cpp

```diff
@@ -348,6 +354,27 @@ std::shared_ptr<asio_ns::ssl::context> SslServerFeature::createSslContext() {
   }
 }
 
+std::shared_ptr<std::vector<std::unique_ptr<asio_ns::ssl::context>>> SslServerFeature::createSslContext() {
```

Comment:
maybe rename to `createSslContexts`?

Reply:
maybe this should be a struct or typedef

Reply:
Done. Both.